packages/api-electron/src/preload/shims.ts

```diff
@@ -1,13 +1,23 @@
 const remote = require('electron').remote;
 const eNotify = remote.require('electron-notify');
+const PERMISSION_GRANTED = "granted";
```

Comment:
perhaps now would be a good time to rename this file from `shims.ts` to something that follows the naming convention for the other files in this folder?